config/build/dpkg_build/install_dlt.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
```

Comment:
Hi Nguyen,
we have no DLT connector in our OSS version,.
Thank you,
 Thomas

Reply:
Hi Thomas,
This is reosolved in commit: 478a4b5224dc82ecdde7bd04339cfda016f33e90